sched/mqueue/mq_initialize.c

```diff
@@ -52,6 +52,22 @@ struct list_node g_msgfreeirq;
 
 #endif
 
+/****************************************************************************
+ * Private Data
+ ****************************************************************************/
+
+/* This is a pool of pre-allocated message queue buffers */
+
+static uint8_t g_msgpool[
```

Comment:
the alignment mayn't match the requirementt of mqueue_msg_s/msgbuf_s

Reply:
Done